gulp/config.js

```diff
@@ -78,6 +78,7 @@ module.exports = {
         // documentation site
         pldoc_src: pldocSrc + '/static/js',
         pldoc_src_files: pldocSrc + '/static/js/**/*.js',
+        pldoc_src_json: pldocSrc + '/static/js/**/*.json',
```

Comment:
Now that your colors file is `js` and brought in through requirejs - it shouldn't need any special treatment - you can remove this and the new Gulp task below